doc/user/content/ingest-data/postgres/amazon-rds.md

```diff
@@ -57,18 +57,18 @@ As a first step, you need to make sure logical replication is enabled.
     - If logical replication is already on, skip to [Create a publication and a
       Materialize user section](#2-create-a-publication-and-a-replication-user).
 
-1. [Create a custom RDS parameter group](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_WorkingWithDBInstanceParamGroups.html#USER_WorkingWithParamGroups.Creating).
+1. [Create a db parameter group in RDS](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_WorkingWithParamGroups.Creating.html) using the AWS management console..
 
     - Set **Parameter group family** to your PostgreSQL version.
     - Set **Type** to **DB Parameter Group**.
+    - Set **Engine type** to PostgreSQL
```

Comment:
Missing item.  It's not obvious since there are an Aurora PostgreSQL option as well and they're sorta trying to pretend they're the same.